templates/common/infra/terraform/core/host/appservice/appservicenode/appservicenode_output.tf

```diff
@@ -0,0 +1,8 @@
+output "DEFAULT_HOST_NAME" {
+  value = azurerm_linux_web_app.web.default_hostname
+}
+
+output "IDENTITY_PRINCIPAL_ID" {

```

Comment:
Outputs are camel in bicep, should we match that here?

Reply:
[Terraform naming convention](https://www.terraform-best-practices.com/naming) for outputs is to use underscores